build/ublue-os-just/00-default.just

```diff
@@ -31,11 +31,10 @@ regenerate-grub:
 
 # Enroll Nvidia driver & KMOD signing key for secure boot - Enter password "universalblue" if prompted
 enroll-secure-boot-key:
-    echo 'Enter password "universalblue" if prompted after your user password.'
     sudo mokutil --timeout -1
+    echo 'Enter password "universalblue" if prompted next'
```

Comment:
Would [L36](https://github.com/Malix-Labs/config/blob/30f286b34eb93173ef4ba4581eefc7a73586a0e6/build/ublue-os-just/00-default.just#L36) always prompt for the MOK password in all cases ?
If yes, then I also suggest this :
```suggestion
    echo 'The next line will prompt for a MOK password. Then, input "universalblue"'
```

Reply:
Your suggestion is good, as yes, `mokutil  --import` will always prompt for a password.